dev/com.ibm.ws.grpc_fat/.classpath

```diff
@@ -3,8 +3,50 @@
 	<classpathentry kind="src" path="fat/src"/>
 	<classpathentry kind="src" path="test-applications/HelloWorldClient.war/src"/>
 	<classpathentry kind="src" path="test-applications/HelloWorldService.war/src"/>
+	<classpathentry kind="src" path="test-applications/FavoriteBeerService.war/src"/>
 	<classpathentry kind="src" path="generated-src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="lib" path="C:/Users/LoriAdington/.m2/repository/com/google/protobuf/protobuf-java/3.11.1/protobuf-java-3.11.1.jar">
```

Comment:
Can you remove these classpath entries that reference files in your local workspace?

Reply:
I've fixed this.